server/routes/feedBackRouter.js

```diff
@@ -0,0 +1,12 @@
+const feedBackRouter = require('express').Router();
+const {
+  getAllFeedBack, addFeedBack, updateFeedBack, deleteFeedBack,
+} = require('../controllers/feedBack');
+const checkAuth = require('../middlewares/checkauth');
+
+feedBackRouter.post('/addFeedback/:commentingId', checkAuth, addFeedBack);
```

Comment:
When u call this router you call it with ```feedback``` prefix,
We will suffice with this prefix, and we do not need other additional words, such as ```addFeedback, allFeedback, deleteFeedback```

Reply:
+1